packages/stateless/components/dao/tabs/AppsTab.tsx

```diff
@@ -75,13 +75,12 @@ const InnerAppsTab = ({
   useEffect(() => {
     try {
       if (iframe && url && new URL(url).href) {
-        iframe.src = url
+        go(url)
```

Comment:
because `go` is redefined on every render (it's not wrapped in a [`useCallback`](https://react.dev/reference/react/useCallback) hook), this is actually going to fire on every render.

what i had in mind instead is that you remove the `if` statement from the `go` function so that it just contains `setUrl(destUrl)`, and then set the `useEffect` dependencies to `[iframe, url]` so that the effect hook sets `iframe.src = url` whenever the `url` changes. does that make sense? idk if it'll fix it but it's worth a shot.

Reply:
Whoops there you go. Tested it and works as intended